README.md

```diff
@@ -113,6 +114,9 @@ See [ROADMAP](/ROADMAP.md).
 
 Chaos Mesh is licensed under the Apache License, Version 2.0. See [LICENSE](/LICENSE) for the full license text.
 
+
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fchaos-mesh%2Fchaos-mesh.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fchaos-mesh%2Fchaos-mesh?ref=badge_large)
```

Comment:
why add it twice?

Reply:
@WangXiangUSTC one is on the top and the other is at the bottom. And the style of them are different.